setup.py

```diff
@@ -55,7 +55,14 @@ def build_cython():
         if os.path.exists(pyx) and needs_compilation(c, pyx)]
     if len(nc_pyx_filenames) > 0:
         cmd = ['cython'] + nc_pyx_filenames
-        subprocess.check_call(cmd)
+        try:
+            subprocess.check_call(cmd)
+        except FileNotFoundError:
+            raise RuntimeError(
+                "Failed to find Cython. "
+                "Please download and install Cython "
+                "and make it available in the path"
+            )
```

Comment:
This is the second segfault I've triggered in conda without an obvious reason.  Does conda just randomly segfault?

The other one: https://github.com/LeeKamentsky/python-javabridge/pull/131/commits/6b9a4da0b440408712f7b098272c0182d29c42e0

Reply:
Looks like this might be a conda bug (via tqdm). tqdm fixed it and released but it appears conda hasn't gotten a release done yet.

https://github.com/conda/conda/issues/7002